mlflow/tracking/client.py

```diff
@@ -142,6 +142,57 @@ def get_metric_history(self, run_id, key):
         :param key: Metric name within the run
 
         :return: A list of :py:class:`mlflow.entities.Metric` entities if logged, else empty list
+
+        .. code-block:: python
+            :caption: Example
+
+            from mlflow.tracking import MlflowClient
+
+            def print_metric_info(history):
+                for m in history:
+                    print("name: {}".format(m.key))
+                    print("value: {}".format(m.value))
+                    print("timestamp: {}".format(m.timestamp))
+                    print("--")
```

Comment:
```suggestion
                    print("name: {}".format(m.key))
                    print("value: {}".format(m.value))
                    print("timestamp: {}".format(m.timestamp))
                    print("value: {}".format(m.step))
                    print("--")
```

to make sure the specifed step is logged?

Reply:
Sure, why not.